warehouse/static/js/warehouse/controllers/increment_controller.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Controller } from "stimulus";
+
+export default class extends Controller {
+  static targets = ["button"];
+
+  // Initially set default to 5 projects visible
+  constructor() {
+    super();
+    this.trending_visible = 5;
+    this.latest_visible = 5;
```

Comment:
I think it might be neater and simpler if instead of having the controller wrapping both columns, having one instance of a controller per column, i.e. the `<div class="col-half">`.

That would make remove all the complexity below on parsing and retrieving the `eventType` and the separate `_visible` variables.

Reply:
You're right, it was hard to read